main.js

```diff
@@ -1,6 +1,6 @@
 const books = document.getElementById('books');
-let titulo = document.getElementById('titulo');
-let autor = document.getElementById('autor');
+const titulo = document.getElementById('titulo');
+const autor = document.getElementById('autor');
 const addBtn = document.getElementById('btn');
 
 let dataBooks = [];
```

Comment:
- [ ] Following the project requirements, you should use  JavaScript classes instead of objects and arrays, please note that you can use objects and arrays but it has to be inside the classes. See the example below:
```javascript
class Books {
  constructor() {
    this.dataBooks = [];
  }
...
}
```